### PR TITLE
Minimal theme based on TruncatedPwd_WindowTitle

### PR DESCRIPTION
**Pull request by imbibinebe**
Hi,
Here is a new theme for users who do not need to remind who they are, neither where their GIT projects are, nor on which machine they work :)
Hope you will like it !

### CODE REVIEW

**Comment by tpraxl** [comment]
+1 very much to the point! I really like that theme.

**Comment by magicmonty** [comment]
Unfortunately, I get an error on Mac OSX 10.11, when I'm not in a git repository:

image

**Comment by tpraxl** [comment]
Didn't notice that, because I set `GIT_PROMPT_ONLY_IN_REPO=1` in `~/.bashrc`

File: themes/Minimal.bgptheme [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+# fi
+#
+# Imbibinebe <imbibinebe@gmail.com> [https://github.com/imbibinebe]
+##############################################################################
+override_git_prompt_colors() {
+  GIT_PROMPT_THEME_NAME="Minimal"
+
+  #Overrides the prompt_callback function used by bash-git-prompt
+  function prompt_callback {
+    GIT_CONTAINER_FOLDER_FULLPATH=$(git rev-parse --show-toplevel)
```
Comment (tpraxl) [inline themes/Minimal.bgptheme]: @magicmonty / @imbibinebe 

This line is likely the cause of https://github.com/magicmonty/bash-git-prompt/pull/248#issuecomment-232460229

I don't know the codebase well enough to repair that myself, but I'm pretty sure that the bug would be fixed if you called `git rev-parse --show-toplevel` only when we're inside a git repo directory.

If that is not possible, maybe @magicmonty could provide a conditional function (e.g. `cond-git` that delegates only to git when we're inside a repo). This way, theme-authors could make use of all git commands. In this case `cond-git rev-parse --show-toplevel`.

Maybe @magicmonty could simply provide a variable like `GIT_TOPLEVEL_FOLDER`. 

As a user, I expect the output to be straight to the point in order to be of good use, so I would very much appreciate this theme.

**Comment by ogr3** [comment]
You need to fix the theme so that it behaves outside a git-repo as well and state what path it should show then. Otherwise it will not work for app people that uses this prompe everywhere.

You can use this trick to check wherer you are in the repo:

` git_root_dir=$(git rev-parse --show-toplevel 2> /dev/null)`

if the variable is not set you should set the path to whatever it should be outside a git repo. If you just use a cond-git it will fail on the row below calling for basename.

**Comment by imbibinebe** [comment]
Hi all. Glad you like that theme :-)
I'll find time to fix it next week, cause I am still in holidays
Thanks

**Comment by imbibinebe** [comment]
This should work now. Tell me

**Comment by ogr3** [comment]
realpath command does not exist in default Ubuntu nor OS X, so the theme will fail on those.

Maybe you could use `$(pwd)` instead?

**Comment by magicmonty** [comment]
Awesome, thanks

**Pull request merged**